src/data_model.py

```diff
@@ -33,10 +33,14 @@ class Camera:
     image_size_x_px: int #Number of pixels in the image along the x axis
     image_size_y_px: int #Number of pixels in the image along the y axis
 
-
+@dataclass
 class Waypoint:
     """
     Waypoints are positions where the drone should fly to and capture a photo.
     """
 
-    pass
+    x: float
+    y: float
+    z: float
+    speed_mps: float
+    photo_trigger: bool = False
```

Comment:
In our actual product, we do have some waypoints where we do not take photos. So this is a good thing to introduce!